ipn/prefs.go

```diff
@@ -414,7 +438,16 @@ func (p *Prefs) Equals(p2 *Prefs) bool {
 		compareIPNets(p.AdvertiseRoutes, p2.AdvertiseRoutes) &&
 		compareStrings(p.AdvertiseTags, p2.AdvertiseTags) &&
 		p.Persist.Equals(p2.Persist) &&
-		p.ProfileName == p2.ProfileName
+		p.ProfileName == p2.ProfileName &&
+		p.AutoUpdate.Equals(p2.AutoUpdate)
+}
+
+func (au AutoUpdatePrefs) Equals(au2 AutoUpdatePrefs) bool {
```

Comment:
do you need an Equals method for some reason? it's fine. just curious.

Reply:
Nope, was just mimicking other code. Removed